src/Cake.Common/Tools/DotNetCore/Restore/DotNetCoreRestoreSettings.cs

```diff
@@ -38,7 +40,8 @@ public sealed class DotNetCoreRestoreSettings : DotNetCoreSettings
 
         /// <summary>
         /// Gets or sets a value indicating whether to display any output.
-        /// </summary>
+        /// </summary>
+        [Obsolete("dotnet cli does not have this option")]
```

Comment:
Can we change this to `.NET CLI does not support this option anymore. Please use DotNetCoreRestoreSettings.Verbosity instead.`